### PR TITLE
Completed task 1 (asp.net)

### PR DESCRIPTION
**Pull request by kuleshovaann**

### CODE REVIEW

File: Delivery-Service.API/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DeliveryService.Contracts;
+using DeliveryService.Database;
+using DeliveryService.Services;
+using DeliveryService.Models;
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Controllers/WeatherForecastController.cs]: add build check in GitHub
close PRs from other modules
remove this default "WeatherForecastController "

Comment (yevhenbilushko) [reply Delivery-Service.API/Controllers/WeatherForecastController.cs]: create your own
or rename it

File: Delivery-Service.API/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        {
+            _logger = logger;
+        }
+
+        // Task 1 ASP.Net
+
+        //  GET/menu
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Controllers/WeatherForecastController.cs]: for dish deleting it is Id need
so please add Id
do u lan to ad dish
if so - add endpoint
for other entitiesas well

File: Delivery-Service.API/Controllers/WeatherForecastController.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            return service.GetMenu();
+
+            //var rng = new Random();
+            //return Enumerable.Range(1, 5).Select(index => new WeatherForecast
+            //{
+            //    Date = DateTime.Now.AddDays(index),
+            //    TemperatureC = rng.Next(-20, 55),
+            //    Summary = Summaries[rng.Next(Summaries.Length)]
+            //})
+            //.ToArray();
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Controllers/WeatherForecastController.cs]: removecomment

File: Delivery-Service.API/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Startup.cs]: remove comment

File: Delivery-Service.API/Startup.cs [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        {
+
+            services.AddControllers();
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "Delivery_Service.API", Version = "v1" });
+            });
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Startup.cs]: remove comment

File: Delivery-Service.API/WeatherForecast.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Delivery_Service.API
+{
+    public class WeatherForecast
```
Comment (yevhenbilushko) [inline Delivery-Service.API/WeatherForecast.cs]: remove this entity

File: Delivery-service/Delivery-Servise.API/Controllers/WeatherForecastController.cs [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Delivery_Servise.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```
Comment (yevhenbilushko) [inline Delivery-service/Delivery-Servise.API/Controllers/WeatherForecastController.cs]: do we need it ?

File: Delivery-service/Delivery-Servise.API/Startup.cs [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Delivery_Servise.API
```
Comment (yevhenbilushko) [inline Delivery-service/Delivery-Servise.API/Startup.cs]: seems like it is duplication
handle it please

File: Delivery-service/Entity/DeliverySystem/Program.cs [hunk 9]
```diff
@@ -9,10 +10,11 @@ class Program
     {
         static void Main(string[] args)
```
Comment (yevhenbilushko) [inline Delivery-service/Entity/DeliverySystem/Program.cs:11]: try to use classlib project type
not console app project type

File: Delivery-service/Models/Services/CompanyServices.cs [hunk 10]
```diff
@@ -58,5 +59,16 @@ public void GetStartMenu(Company company)
 
             _serializator.SaveToFlieDataCompany(company);
         }
+
+        //mothod for TASK 1 ASP
```
Comment (yevhenbilushko) [inline Delivery-service/Models/Services/CompanyServices.cs]: remove comment

File: Delivery-Service.API/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (kuleshovaann) [reply Delivery-Service.API/Controllers/WeatherForecastController.cs]: how to add build check?

File: Delivery-service/Entity/DeliverySystem/Program.cs (hunk 9 above)
Comment (kuleshovaann) [reply Delivery-service/Entity/DeliverySystem/Program.cs:11]: don't understand how to use classlib project type

Comment (yevhenbilushko) [reply Delivery-service/Entity/DeliverySystem/Program.cs:11]: image

Comment (yevhenbilushko) [reply Delivery-service/Entity/DeliverySystem/Program.cs:11]: image

Comment (yevhenbilushko) [reply Delivery-service/Entity/DeliverySystem/Program.cs:11]: in properties

File: Delivery-Service.API/Controllers/DishController.cs [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        [HttpGet]
+        public IEnumerable<Dish> Get()
+        {
+            var service = new DeliveryService.Services.CompanyServices(new OrderDatabase(new FileServices()), new Logger(), new FileServices());
+            return service.GetMenu();
+        }
+    }
+
+    //GET/menu
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Controllers/DishController.cs]: remove comments

File: Delivery-Service.API/Delivery-Service.API.csproj [hunk 12]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RootNamespace>Delivery_Service.API</RootNamespace>
+    <OutputType>Exe</OutputType>
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Delivery-Service.API.csproj]: switch to class lib project type

File: Delivery-Service.API/Startup.cs [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+
```
Comment (yevhenbilushko) [inline Delivery-Service.API/Startup.cs]: remove extra line

File: Delivery-service/Delivery-Servise.API/Startup.cs [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
```
Comment (yevhenbilushko) [inline Delivery-service/Delivery-Servise.API/Startup.cs]: remove comments

File: Delivery-service/Delivery-Servise.API/Startup.cs [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+        {
+
+            services.AddControllers();
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "Delivery_Servise.API", Version = "v1" });
+            });
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
```
Comment (yevhenbilushko) [inline Delivery-service/Delivery-Servise.API/Startup.cs]: remove comments

File: Delivery-service/Delivery-Servise.API/WeatherForecast.cs [hunk 16]
```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Delivery_Servise.API
+{
+    public class WeatherForecast
```
Comment (yevhenbilushko) [inline Delivery-service/Delivery-Servise.API/WeatherForecast.cs]: remove it

File: Delivery-service/Entity/DeliverySystem/Program.cs [hunk 17]
```diff
@@ -9,10 +10,11 @@ class Program
...
         {
+            var data = new DataContext();
             var providerDataBase = new ProviderDataBase();
             var productDataBase = new ProductDataBase();
             var creator = new DataCreator(providerDataBase, productDataBase);
-            var linq = new LinqRequests(providerDataBase, productDataBase);
+            //var linq = new LinqRequests(providerDataBase, productDataBase);
```
Comment (yevhenbilushko) [inline Delivery-service/Entity/DeliverySystem/Program.cs]: remove it or use it

File: Delivery-Service.API/Delivery-Service.API.csproj (hunk 12 above)
Comment (kuleshovaann) [reply Delivery-Service.API/Delivery-Service.API.csproj]: if i set Delivery_Sevice.API as class lib, i have error "unble to run your project. the RunCommand property is not defined"

Comment (yevhenbilushko) [reply Delivery-Service.API/Delivery-Service.API.csproj]: ok

File: Delivery-service/Delivery-Servise.API/Startup.cs (hunk 14 above)
Comment (yevhenbilushko) [reply Delivery-service/Delivery-Servise.API/Startup.cs]: remove it

File: Delivery-service/Delivery-Servise.API/Startup.cs (hunk 15 above)
Comment (yevhenbilushko) [reply Delivery-service/Delivery-Servise.API/Startup.cs]: remove it

File: Delivery-service/Delivery-Servise.API/WeatherForecast.cs (hunk 16 above)
Comment (yevhenbilushko) [reply Delivery-service/Delivery-Servise.API/WeatherForecast.cs]: remove it

**Review by yevhenbilushko — APPROVED** [review]

**Comment by yevhenbilushko** [comment]
fix and merge

**Comment by yevhenbilushko** [comment]
and fix build
cause build check failed

**Pull request merged**